changelog/fragments/3173-bugfix.yaml

```diff
@@ -0,0 +1,10 @@
+entries:
+  - description: >
+      Prevent the ansible-operator from mangling variables containing the uppercase representations of special words (e.g IP, HTTP, etc).
+    kind: change
+    breaking: true
+
+    migration:
+      header: This new change will fix the ansible variables that are getting mangled
+      body: >
+        _/operator-sdk/pkg/ansible/paramconv/paramconv.go will be able to handle variables properly (Camel and Snake case conversion)
```

Comment:
This should contain the instructions for a user to avoid being broken by this change. It shouldn't mention any internal code (users don't know or care about this file), and it should have a few examples of how variables will be handled differently